conans/test/command/build_test.py

```diff
@@ -63,7 +63,7 @@ def build(self):
 """
         client.save({CONANFILE: conanfile_user_info}, clean_first=True)
         client.run("install --build=missing")
-        client.run("build .")
+        client.run("build ./conanfile.py")
```

Comment:
Some some with ``build .``

Reply:
Yes, I left some tests with just `build .` Check line 110 for example